prog/kubernetes/kubernetes_cluster_nexus.rb

```diff
@@ -61,6 +62,7 @@ def before_run
       dst_port: 6443,
       health_check_endpoint: "/healthz",
       health_check_protocol: "tcp",
+      custom_hostname_dns_zone_id: custom_dns_zone_id,
```

Comment:
you also have to provide `custom_hostname_prefix`. Without it, the LB doesn't use the customer hostname for the DNS record and the DNS zone sync fails.

Reply:
I tried once and caused an incident, so that's what I thought. Maybe it was me doing something wrong.